vignettes/projecting_incidence.Rmd

```diff
@@ -0,0 +1,292 @@
+---
+title: "Projecting infectious disease incidence using early outbreak data"
+author: "James Azam, Sebastian Funk"
+output:
+  bookdown::html_vignette2:
+    fig_caption: yes
+    code_folding: show
+pkgdown:
+  as_is: true
+bibliography: references.bib
+link-citations: true
+vignette: >
+  %\VignetteIndexEntry{Projecting infectious disease incidence using early outbreak data}
+  %\VignetteEncoding{UTF-8}
+  %\VignetteEngine{knitr::rmarkdown}
+editor_options: 
+  chunk_output_type: console
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE, 
+                      message = FALSE, 
+                      warning = FALSE, 
+                      collapse = TRUE,
+                      comment = "#>",
+                      dpi = 300
+                      )
+
+```
+
+## Overview
+
+Branching processes can be used to project infectious disease trends provided 
+we can characterize the distribution of times between 
+successive cases (serial interval), and the distribution of secondary cases 
+produced by a single individual (offspring distribution). Such simulations can 
+be achieved in `bpmodels` with the `chain_sim()` function. @Pearson2020, and 
+@abbott2020 illustrate its application to COVID-19. 
+
+The purpose of this vignette is to use early data on COVID-19 in South Africa 
+[@marivate2020] to illustrate how `bpmodels` can be used to forecast an 
+outbreak. 
+
+
+Let's load the required packages
+
+```{r packages, include=TRUE}
+library("bpmodels")
+library("dplyr")
+library("ggplot2")
+library("lubridate")
+```
+
+## Data
+
+We will get and clean the first $15$ days of the COVID-19 
+outbreak in South Africa to seed the simulation for this example.
```

Comment:
In this section I would remove the all data wrangling code (as it is a copy of the data-raw script) and just call `data()`. You can provide a link to the data-raw script if users would like to see how the data is generated. Currently `data()` is not called anywhere in the package.

Reply:
I have switched to calling data(...).